src/python/BuildingControlsSimulator/DataClients/WeatherChannel.py

```diff
@@ -28,6 +28,7 @@
 @attr.s(kw_only=True)
 class WeatherChannel(DataChannel):
     """Client for weather data."""
+    fill_nsrdb_data = attr.ib(default=None)
```

Comment:
the fill data shouldn't persist in the instances, it should only exist during the fill function called from `DataClient.get_data()`

Reply:
sure, makes sense.